terraform/projects/pr-dashboard/rds.tf

```diff
@@ -0,0 +1,88 @@
+
+resource "aws_db_instance" "rds" {
+  identifier = "web-platform-tests-db"
+  storage_type = "gp2"
+  allocated_storage = 10
+  engine = "postgres"
+  engine_version = "9.5.4"
```

Comment:
You can use 9.6.3-R1 If you'd like here, that's the latest on RDS right now.

Reply:
matching the release on the server for now, but yeah.